scripts/stujo_etl_staging.sh

```
@@ -0,0 +1,108 @@
+    echo "       Check 'gcloud auth list' and that you have secretAccessor on it." >&2
+    exit 1
+  fi
+  if [[ -z "${val}" ]]; then
+    echo "ERROR: secret '${name}' is empty." >&2
+    exit 1
+  fi
+  printf '%s' "${val}"
+}
+
+echo "Fetching staging secrets from Secret Manager (${GCP_PROJECT})…"
+export HASURA_ADMIN_SECRET="$(fetch_secret hasura-graphql-admin-key)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)